_spinnaker/terraform_integration.md

```diff
@@ -4,114 +4,113 @@ title: Terraform Integration
 order: 141
 ---
 
-The following tutorial will walk you through how to setup our Terraform integration and execute Terraform code stored in a Github repo as part of a Spinnaker pipeline. We'll assume that you're using Terraform to create and manage infrastructure on AWS.
-
 Note that the Terraform integration is in beta while we work on improving the user experience.
 
+{:.no_toc}
+## Overview
+
+Use Armory Spinnaker's Terraform Integration to integrate your infrastructure-as-code with Terraform workflow into your Spinnaker instance. Manage your application infrastructure as part of a Spinnaker pipeline. The following tutorials walk you through how to setup Armory's Terraform integration and execute Terraform code stored in a Git repo as part of a Spinnaker pipeline. The examples on this page describe a workflow for using Terraform to create and manage infrastructure on AWS. 
+
```

Comment:
^^ Small nit - `as part of a Spinnaker pipeline` is used in two consecutive sentences. Feels a bit repetitive. Perhaps eliminate the first sentence?